### PR TITLE
docs(id): Separate the CONTRIBUTING.md file based on ID and EN translation

### LINKED ISSUE #319: Separate the CONTRIBUTING.md file based on ID and EN translation
Separate the CONTRIBUTING.md file into `CONTRIBUTING_ID.md` and `CONTRIBUTING.md` , considering that the majority of contributors here are Indonesian and we cannot translate them into languages other than those two languages.

the benefit of this is that it makes it easier to read the existing guide.

## Tasks

- [ ] Create a `CONTRIBUTING_ID.md` file.
- [ ] Move all the Indonesian language content of `CONTRIBUTING.md` file to the newly-created file.

### PR DESCRIPTION
**Pull request by Namchee**
Closes #319 

## Description

Separates EN and ID contribution guidelines to their own markdown files and add a separate link in the main README file. 

## Current Tasks

- [x] Create a CONTRIBUTING_ID.md file.
- [x] Move all the Indonesian language content of CONTRIBUTING.md file to the newly-created file.

### CODE REVIEW

**Review by zainfathoni — CHANGES_REQUESTED** [review]
This small `contributing guidelines` link is automatically linked to the `CONTRIBUTING.md` file.

Screenshot 2021-07-23 at 19 12 33

If we split it and ended up with no `CONTRIBUTING.md` file at all, would we still have that link automatically created by GitHub, @Namchee?
If not, I would suggest that we keep one of them (preferably the English version) as `CONTRIBUTING.md` file, then for the Bahasa Indonesia version, we can name it whatever we want, such as `CONTRIBUTING_ID.md` like you have implemented.

**Review by resir014 — CHANGES_REQUESTED** [review]
Please keep the `CONTRIBUTING_EN.md` filename as `CONTRIBUTING.md`. Otherwise it won't be picked up by the GitHub UI when people create an issue/PR.

image

**Comment by zainfathoni** [comment]
We requested the same changes with only a two-minute difference, @resir014!
🤣

File: CONTRIBUTING_ID.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+## Cara berkontribusi
```
Comment (resir014) [inline CONTRIBUTING_ID.md:1]:
```suggestion
# Berkontribusi ke WargaBantuWarga

## Cara berkontribusi
```

@Namchee Always start a Markdown document with h1. We can use the same heading from the English document.

**Review by resir014 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING_ID.md (hunk 1 above)
Comment (Namchee) [reply CONTRIBUTING_ID.md:3]: Solved by ab1903a2ba95a9b39c9afe7594a16ca5a630ed75

**Review by resir014 — APPROVED** [review]
🚢

**Review by zainfathoni — APPROVED** [review]
LGTM 💯 

Thanks, @Namchee for contributing to the docs. cc @all-contributors

**Comment by zainfathoni** [comment]
@all-contributors please add @Namchee for doc

**Pull request merged**